### PR TITLE
Refactor peer store

### PR DESCRIPTION
**Pull request by jjyr**
1. change modules layout.
2. remove unnecessary code.
3. enable file storage mode.

### CODE REVIEW

File: network/src/peer_store/score.rs [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+        let schema = [
+            (Behaviour::FailedToConnect, -20),
+            (Behaviour::FailedToPing, -10),
+            (Behaviour::Ping, 5),
+            (Behaviour::Connect, 10),
+            (Behaviour::UnexpectedDisconnect, -20),
+        ]
+        .iter()
+        .cloned()
+        .collect();
```
Comment (TheWaWaR) [inline network/src/peer_store/score.rs]: `vec![Key, Value].into_iter().collect();`

**Comment by TheWaWaR** [comment]
PeerStore API changed, but `service/discovery_service.rs` not changed.
Do not understand why CI passed.

**Comment by jjyr** [comment]
bors try

**Comment by TheWaWaR** [comment]
@jjyr 
You can run `make ci` in your machine, then push.

**Pull request merged**